city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,143 @@
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookJusticeAdvisorySpider(CityScrapersSpider):
+    name = "cook_justice_advisory"
+    agency = "Cook County Justice Advisory"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cookcountyil.gov"]
+
+    def start_requests(self):
+        url = "https://www.cookcountyil.gov/service/justice-advisory-council-meetings"
+        yield scrapy.Request(url=url, method="GET", callback=self.parse)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url, callback=self._parse_event, dont_filter=True)
+
+    def _parse_event(self, response):
+        """Parse the event page."""
+        title = self._parse_title(response)
+        meeting = Meeting(
+            title=title,
+            description=self._parse_description(response),
+            classification=ADVISORY_COMMITTEE,
+            start=self._parse_start(response),
+            end=self._parse_end(response),
+            time_notes="",
+            all_day=self._parse_all_day(response),
+            location=self._parse_location(response),
+            links=self._parse_links(response),
+            source=response.url,
+        )
+        meeting["id"] = self._get_id(meeting)
+        meeting["status"] = self._get_status(meeting)
+        return meeting
+
+    def _get_event_urls(self, response):
```

Comment:
The wording for the meetings is sometimes inconsistent, so searching for "JAC Council" might not always work. We'll also want to get some recent past meetings as well as more than the next two upcoming meetings if present, so we should probably iterate through the calendar page here

Reply:
Since "JAC Council" might not always work, what would you suggest searching for when iterating through the calendar page?

Reply:
Checking "JAC" or "Justice Advisory" after extracting the text from a link and cleaning up extra spaces is probably safest. Using the selector for text might not work in some cases where there's extra text inside accidentally

Reply:
I still think it would be safer to pull the string out of the selector and do multiple comparisons for "JAC" as well as "Justice Advisory" in case they change this up since using the text contains selector is pretty strict

Reply:
I got the event title out of the selector and put some conditions/comparisons in there. Are there any more comparisons that might be useful to add? I wrote the code so that adding more comparisons is as simple as possible.